app/javascript/gobierto_dashboards/webapp/Maker.vue

```diff
@@ -39,7 +39,7 @@
           <SmallCard
             v-for="({ name }, key) in cards"
             :key="key"
-            :name="name"
+            :name="name()"
```

Comment:
why is it a function call?

Reply:
OK, it comes from Widget, right?

Reply:
Exactly. I need to call `I18n.t()` after mounted in order to solve properly the displayed language.